src/plugins/intel_cpu/src/nodes/executors/acl/acl_ie_scheduler.cpp

```diff
@@ -51,7 +51,7 @@ void ACLScheduler::schedule_custom(ICPPKernel* kernel, const Hints& hints, const
         const unsigned int num_iterations = m * n;
         const auto num_threads = get_num_threads(num_iterations);
 
-        unsigned m_threads, n_threads;
+        unsigned m_threads = 0, n_threads = 0;
```

Comment:
Can be  replaced with structured binding

Reply:
Done